pages/public_cloud/compute/transfer_instance_backup_from_one_datacentre_to_another/guide.en-gb.md

```diff
@@ -103,6 +103,23 @@ To transfer the backup to the new OpenStack region, use this command:
 $ openstack image create --disk-format qcow2 --container-format bare --file snap_server1.qcow snap_server1
 ```
 
+> [!warning]
+>
+> If your instance uses a Windows image, you need to add specific properties. Otherwise, when creating the instance via the manager, it will not be possible to associate a flavor of the win-x-x type. This type of flavor, and only this one, enables authentication to the [KMS OVH](https://help.ovhcloud.com/csm/en-gb-dedicated-servers-windows-key?id=kb_article_view&sysparm_article=KB0044091).
```

Comment:
```suggestion
> If your instance uses a Windows image, you need to add specific properties. Otherwise, when creating the instance via the OVHcloud Control Panel, it will not be possible to associate a flavor of the win-x-x type. This type of flavor, and only this one, enables authentication to the [OVHcloud KMS](/pages/bare_metal_cloud/dedicated_servers/windows_key).
```